example https-nginx make fail.

I follow [https-nginx](https://github.com/kubernetes/kubernetes/tree/master/examples/https-nginx) to create a nginx example. And I got stuck at the beginning.
```
$ make keys secret KEY=/tmp/nginx.key CERT=/tmp/nginx.crt SECRET=/tmp/secret.json
# The CName used here is specific to the service specified in nginx-app.yaml.
openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/nginx.key -out /tmp/nginx.crt -subj "/CN=nginxsvc/O=nginxsvc"
Generating a 2048 bit RSA private key
....+++
...........+++
writing new private key to '/tmp/nginx.key'
-----
CGO_ENABLED=0 GOOS=linux go run -a -installsuffix cgo -ldflags '-w' make_secret.go -crt /tmp/nginx.crt -key /tmp/nginx.key > /tmp/secret.json
panic: The legacy v1 API is not registered. Please check the KUBE_API_VERSIONS environment variable.

goroutine 1 [running]:
k8s.io/kubernetes/pkg/api/latest.GroupMetaMap.GroupOrDie(0xc20803cab0, 0x0, 0x0, 0xc2080387e0)
        /root/workspace/kubernetes/src/k8s.io/kubernetes/pkg/api/latest/latest.go:76 +0xd4
k8s.io/kubernetes/pkg/api/latest.GroupMetaMap.GroupOrDie·fm(0x0, 0x0, 0xc2080cbf28)
        /root/workspace/kubernetes/src/k8s.io/kubernetes/pkg/api/latest/latest.go:36 +0x3e
main.main()
        /root/workspace/kubernetes/src/k8s.io/kubernetes/examples/https-nginx/make_secret.go:66 +0x288

goroutine 5 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0xaf9fc0)
        /root/workspace/kubernetes/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x78
created by github.com/golang/glog.init·1
        /root/workspace/kubernetes/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2a7
exit status 2
make: *** [secret] Error 1
```
@bprashanth @caesarxuchao

--- a/examples/https-nginx/README.md
+++ b/examples/https-nginx/README.md
@@ -38,7 +38,7 @@ It uses an [nginx server block](http://wiki.nginx.org/ServerBlockExample) to ser
 
 ### Generate certificates
 
-First generate a self signed rsa key and certificate that the server can use for TLS.
+First generate a self signed rsa key and certificate that the server can use for TLS. This step invokes the make_secret.go script in the same directory, which uses the kubernetes api to generate a secret json config in /tmp/secret.json.
 
 ```sh
 $ make keys secret KEY=/tmp/nginx.key CERT=/tmp/nginx.crt SECRET=/tmp/secret.json

--- a/examples/https-nginx/make_secret.go
+++ b/examples/https-nginx/make_secret.go
@@ -29,6 +29,9 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/latest"
 	"k8s.io/kubernetes/pkg/runtime"
+
+	// This installs the legacy v1 API
+	_ "k8s.io/kubernetes/pkg/api/install"
 )
 
 // TODO:

